### ISSUE TITLE
Logits seems to be missing

### ISSUE THREAD

**Issue by fzyzcjy** [linkedin/Liger-Kernel#328]
### 🐛 Describe the bug

Hi thanks for the library! It seems that the `output.logits` is missing. Looking at code https://github.com/linkedin/Liger-Kernel/blob/6cdc93deee15ab6c843149d6ed660c297c5c2d4a/src/liger_kernel/transformers/model/llama.py#L100, seems that it is not there.

My use case is to get the token_accuracy during training. For example, https://platform.openai.com/docs/guides/fine-tuning/use-a-checkpointed-model OpenAI's fine-tuning also provides this train_mean_token_accuracy metric.

### Versions

latest

**Comment by ByronHsu** [comment]
this is intended by default because FLCE uses the trick to not materialize the logits to save memory. If you want to keep the logits you can set `cross_entropy=True` and `fused_cross_entropy=False` https://github.com/linkedin/Liger-Kernel/blob/6cdc93deee15ab6c843149d6ed660c297c5c2d4a/src/liger_kernel/transformers/monkey_patch.py#L54

**Comment by fzyzcjy** [comment]
I see, thank you!

Then is it possible to not materialize the logits, but do get the max-logit tokens?

**Comment by ByronHsu** [comment]
hmm that is not possible for now, but you can use `cross_entropy=True` to extract the logits. it will also provide mem reduction

**Comment by fzyzcjy** [comment]
Get it, thanks! Btw, currently transformers Trainer only has a bool flag to use liger kernel. Thus it would be great to allow it to pass some extra kwargs. For example, `TrainingArguments(liger_kernel_kwargs=dict(cross_entropy=True))`, which is passed down to `apply_liger_kernel_to_llama`, and thus enabling this.
